cmd/tsconnect/README.md

```diff
@@ -28,3 +28,13 @@ To serve them, run:
 ```
 
 By default the build output is placed in the `dist/` directory and embedded in the binary, but this can be controlled by the `-distdir` flag. The `-addr` flag controls the interface and port that the serve listens on.
```

Comment:
Does this change deprecate the use of the `dist` folder? Or is that still helpful for local testing?

Reply:
`dist/` is still used when building the app (what's served at https://tsconnect.fly.dev/). That still seems useful as a testbed (that doesn't depend on control), so I'm keeping it around for now.